rest/route/build_task.go

```diff
@@ -105,17 +106,20 @@ func (tbh *tasksByBuildHandler) Run(ctx context.Context) gimlet.Responder {
 
 	tasks = tasks[:lastIndex]
 	for i := range tasks {
+		taskCtx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
Can we add a comment explaining why we're (temporarily or permanently) ignoring the context and link the ticket? Ignoring the context is generally a code smell (or in this case, a temporary bug fix), so it's recommended to add a rationale.